### DISCUSSION TITLE
converting problem with docker command

### DISCUSSION THREAD

**Discussion by JKSTAFF** [containers/podman#24602] [Q&A] [answered]
How to convert
```
docker run -d \
  -v /etc/sing-box:/etc/sing-box/ \
  --name=sing-box \
  --restart=always \
  ghcr.io/sagernet/sing-box \
  -D /var/lib/sing-box \
  -C /etc/sing-box/ run
```
to podman run? Especially the last two line

**Comment by rhatdan** [comment]
The last two lines should be handed to ghcr.io/sagernet/sing-box  so this should just work.  What error are you seeing?

**Reply by JKSTAFF** [reply]
Just confusing, can I write like this ?  
`podman create -it --name singbox --restart=unless-stopped --label io.containers.autoupdate=image --network=host --volume /etc/sing-box:/etc/sing-box:z ghcr.io/sagernet/sing-box:latest -D /var/lib/sing-box -C /etc/sing-box/ run`

**Reply by rhatdan** [reply, marked as answer]
Not unless /etc/sing-box exists on your host system.  Podman does not automatically create files/directories like Docker does.  We consider this a potential security risk.

sudo mkdir /etc/sing-box
sudo podman create -it --name singbox --restart=unless-stopped --label io.containers.autoupdate=image --network=host --volume /etc/sing-box:/etc/sing-box:z ghcr.io/sagernet/sing-box:latest -D /var/lib/sing-box -C /etc/sing-box/ run

Should work.
